layers/cmd/pipeline.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"io"
+	"path/filepath"
+
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
+	"github.com/cloudprivacylabs/opencypher/graph"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	ingestCmd.AddCommand(pipelineCmd)
+	pipelineCmd.Flags().String("file", "", "Pipeline build file")
```

Comment:
Let's have these flags:
 --file: pipeline file. cmdutil.ReadJSONOrYAML this file, parse it as []Step
 --initialGraph: The initial graph, If given, read it into the pipeline context.graph
pipelineCmd should get 1 or more args. Those will be the input files.
Instead of pipelineContext.Input as io.Reader, let's have pipelineContext.InputFiles []string, and set it from args.